tests/buildx/basic.bats

```diff
@@ -58,6 +58,11 @@ EOF
 # buildkit runs at L1 level).
 @test "L0 buildx + L1 buildkit container (docker-container driver)" {
 
+  docker buildx version
+  if [ "$status" -ne 0 ]; then
```

Comment:
If buildx is not supported then we're probably dealing with a relatively-old docker version, which makes me wonder if big improvements like [this](https://github.com/nestybox/sysbox/blob/master/docs/user-guide/configuration.md#speeding-up-sysbox-by-disallowing-trusted-overlay-xattributes) one is supported in amazon-linux.

Reply:
It's not that old; Amazon Linux comes with Docker 20.10, but it does not support `docker buildx`. I tried installing the latest Docker following the official Docker installation instructions, but it's not supported on AL2.